acceptance_tests/features/steps/receipt.py

```diff
@@ -8,22 +9,23 @@
 
 @step("a receipt message is published to the pubsub receipting topic")
 def send_receipt(context):
-    _publish_object_finalize(questionnaire_id=context.emitted_uacs[0]['questionnaireId'])
-
-
-def _publish_object_finalize(case_id="0", tx_id="3d14675d-a25d-4672-a0fe-b960586653e8", questionnaire_id="0"):
-    data = json.dumps({
-        "timeCreated": "2008-08-24T00:00:00Z",
-        "metadata": {
-            "case_id": case_id,
-            "tx_id": tx_id,
-            "questionnaire_id": questionnaire_id,
+    message = json.dumps({
+        "event": {
+            "type": "RESPONSE_RECEIVED",
+            "source": "RH",
+            "channel": "RH",
+            "dateTime": "2021-06-09T14:10:11.910719Z",
+            "transactionId": str(uuid.uuid4())
+        },
+        "payload": {
+            "response": {
+                "questionnaireId": context.emitted_uacs[0]['questionnaireId'],
+                "dateTime": "2021-06-09T14:10:11.909472Z"
+            }
```

Comment:
I think this highlights a bit of feature regression, we can no longer cope with eQ's method of receipting. Not blocking on it, but we should note somewhere that we'll need to build another adapter to make that work again if they use the same receipting method as before.